### PR TITLE
update readme - install docs now in docs repo

### PR DESCRIPTION
**Pull request by sdwheeler**
The Install docs for version 6 have been move to the Docs repo. Removed the installation folder from this repo and updated README.md to point to new published location.

### CODE REVIEW

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
some of the links (like Ubuntu) currently return 404, when will the install docs be updated and published so the links work correctly?

**Comment by sdwheeler** [comment]
@SteveL-MSFT I had a typo in the links. Fixed that. Should be good now.

File: README.md [hunk 1]
```diff
@@ -78,26 +78,25 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
...
+[in-deb9]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#debian-9
+[in-centos]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#centos-7
+[in-rhel7]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#red-hat-enterprise-linux-rhel-7
+[in-opensuse422]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#opensuse-422
+[in-fedora25]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#fedora-25
+[in-fedora26]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#fedora-26
+[in-archlinux]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#arch-linux
+[in-appimage]: https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos-and-linux#linux-appimage
 [in-macos]: https://github.com/PowerShell/PowerShell/tree/master/docs/installation/macos.md
```
Comment (TravisEz13) [inline README.md]: mac link wasn't updated

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
please fix mac link

**Comment by sdwheeler** [comment]
@TravisEz13 Sorry about that. Fixed in latest commit.

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by TravisEz13** [comment]
I filed https://github.com/PowerShell/PowerShell-Docs/issues/2166 . We should have this implemented soon after we make the switch.

**Comment by TravisEz13** [comment]
Need to verify that these changes have been made to the docs:
- [ ] https://github.com/PowerShell/PowerShell/pull/6122
- [ ] https://github.com/PowerShell/PowerShell/pull/5773
- [ ] https://github.com/PowerShell/PowerShell/pull/5823 (verified not in the docs repo)

**Comment by daxian-dbw** [comment]
I quickly compared the setup docs at [powershell-docs](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/docs-conceptual/setup) with the ones here, and it seems they are not in sync:
1. The docs about Linux and Mac has been split into different markdown files since Jan 9, but in powershell-docs they are still in the same file.
2. The powershell core version used in those files in this repository is `6.0.1`, but in powershell-docs, it's `6.0.0`.
3. The content about macOS is not in sync with the mac doc in this repo.

I think we need to make sure the docs in PowerShell-Docs are in sync with the ones in this repository before removing those from this repo.

**Review by daxian-dbw — CHANGES_REQUESTED** [review]
I quickly (_there may be other differences if looking closely_) compared [the setup docs at powershell-docs](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/docs-conceptual/setup) with the ones here, and it seems they are not in sync:
1. The docs about Linux and Mac has been split into different markdown files since Jan 9, but in powershell-docs they are still in the same file.
2. The powershell core version used in those files in this repository is `6.0.1`, but in powershell-docs, it's `6.0.0`.
3. The content about macOS is not in sync with the mac doc in this repo.

I think we need to make sure the docs in PowerShell-Docs are in sync with the ones in this repository before removing those from this repo.

**Pull request closed without merging**